lua-resty-session-dev-1.rockspec

```diff
@@ -22,7 +22,8 @@ build = {
     ["resty.session"] = "lib/resty/session.lua",
     ["resty.session.dshm"] = "lib/resty/session/dshm.lua",
     ["resty.session.file"] = "lib/resty/session/file.lua",
-    ["resty.session.file-thread"] = "lib/resty/session/file-thread.lua",
+    ["resty.session.file.file-thread"] = "lib/resty/session/file/file-thread.lua",
+    ["resty.session.file.file-utils"] = "lib/resty/session/file/file-utils.lua",
```

Comment:
Because these are already in `file` directory, shall we call them:

```suggestion
    ["resty.session.file.thread"] = "lib/resty/session/file/thread.lua",
    ["resty.session.file.utils"] = "lib/resty/session/file/utils.lua",
```

Reply:
done